src/Generators/Po.php

```diff
@@ -12,18 +12,8 @@ public static function toString(Translations $translations)
     {
         $lines = array('msgid ""', 'msgstr ""');
 
-        $headers = array_replace(array(
-            'Project-Id-Version' => '',
-            'Report-Msgid-Bugs-To' => '',
-            'Last-Translator' => '',
-            'Language-Team' => '',
-            'MIME-Version' => '1.0',
-            'Content-Type' => 'text/plain; charset=UTF-8',
-            'Content-Transfer-Encoding' => '8bit',
-            'Language' => $translations->getLanguage(),
-        ), $translations->getHeaders());
-
-        $headers['POT-Creation-Date'] = $headers['PO-Revision-Date'] = date('c');
+        $headers = $translations->getHeaders();
+        $headers['PO-Revision-Date'] = date('c');
```

Comment:
Since Translations may be a .po and not a .pot file, we shouldn't change the 'POT-Creation-Date' header while we save the file.
'POT-Creation-Date' is initialized to the current date when we initialize the Translations instance, but can be overwritten by the .po/.mo extractors.